packages/starlight/translations/hi.json

```diff
@@ -0,0 +1,26 @@
+{
+  "skipLink.label": "इसे छोड़कर कंटेंट पर जाएं",
+  "search.label": "खोजें",
+  "search.shortcutLabel": "(खोजने के लिए / दबाएँ)",
+  "search.cancelLabel": "रद्द करे",
+  "search.devWarning": "खोज केवल उत्पादन बिल्ड में उपलब्ध है। \nस्थानीय स्तर पर परीक्षण करने के लिए साइट बनाए और उसका पूर्वावलोकन करने का प्रयास करें।",
+  "themeSelect.accessibleLabel": "थीम चुनें",
+  "themeSelect.dark": "अँधेरा",
+  "themeSelect.light": "रोशनी",
+  "themeSelect.auto": "स्वत:",
+  "languageSelect.accessibleLabel": "भाषा चुने",
+  "menuButton.accessibleLabel": "मेन्यू",
+  "sidebarNav.accessibleLabel": "मुख्य",
+  "tableOfContents.onThisPage": "इस पृष्ठ पर",
+  "tableOfContents.overview": "अवलोकन",
+  "i18n.untranslatedContent": "यह कंटेंट अभी तक आपकी भाषा में उपलब्ध नहीं है.",
+  "page.editLink": "पृष्ठ संपादित करें",
+  "page.lastUpdated": "आखरी अद्यतन:",
```

Comment:
Feel free to ignore this — I have no way of knowing if it’s correct — but when checking these with Google Translate, it wanted to change this:
```suggestion
  "page.lastUpdated": "आखिरी अद्यतन:",
```